examples/cdn/README.md

```diff
@@ -17,16 +17,17 @@ open index.html
 
 ## The idea behind the example
 
-You can start using Material-UI with minimal Front-end infrastructure,
-which is great for prototyping. We discourage using this approach in production though -
-the client has to download the entire library, regardless of which components are actually used,
-affecting performance and bandwidth utilisation.
+You can start using Material-UI with minimal Front-end infrastructure, which is great for prototyping.
+We discourage using this approach in production, though.
+The client has to download the entire library, regardless of which components are used, affecting performance and bandwidth usage.
 
-[The live preview.](https://combinatronics.com/mui-org/material-ui/master/examples/cdn/index.html)
+<!-- #default-branch-switch -->
+
+[The live preview.](https://combinatronics.com/mui-org/material-ui/next/examples/cdn/index.html)
 
 ## UMD releases
 
 We are providing two Universal Module Definition (UMD) files:
 
-- one for development: https://unpkg.com/@material-ui/core@latest/umd/material-ui.development.js
-- one for production: https://unpkg.com/@material-ui/core@latest/umd/material-ui.production.min.js
+- one for development: https://unpkg.com/@material-ui/core@next/umd/material-ui.development.js
+- one for production: https://unpkg.com/@material-ui/core@next/umd/material-ui.production.min.js
```

Comment:
`<!-- #default-branch-switch -->` here as well

Reply:
Ok, better to be safe

```suggestion
- one for production: https://unpkg.com/@material-ui/core@next/umd/material-ui.production.min.js

<!-- #default-branch-switch -->
```

Reply:
> Ok, better to be safe

We do this everywhere else. Not sure why you think that comment is productive. Or do you believe that you are better than all the other callsites where we did add it?

Reply:
> We do this everywhere else

I guess it depends on who. I didn't add the comment everywhere in the past. My assumption is that it's here to signal that we need to pay attention to the code close to it, and to the rest of the page.